qutebrowser/browser/hints.py

```diff
@@ -19,9 +19,11 @@
 
 """A HintManager to draw hints over links."""
 
-import math
-import functools
 import collections
+import functools
+import math
+import re
+from string import ascii_lowercase
```

Comment:
nitpick: Please only `import string` here and use `string.ascii_lowercase` - I really like to see what module stuff is coming from without looking at the imports.